packages/components/src/Notification/GlobalNotification/_docs/GlobalNotification.mdx

```diff
@@ -0,0 +1,34 @@
+import { Canvas, Controls, Meta, Description} from "@storybook/blocks"
+import { Installation, KaioNotification, ResourceLinks } from "~storybook/components"
+import * as GlobalNotificationExamples from "./GlobalNotification.stories"
+
+<Meta title="Components/Notifications/Global Notification" />
+
+# GlobalNotification
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Notification/GlobalNotification"
```

Comment:
Where did the `figma` and `designGuidelines` keys go?

Reply:
Must of lost them in shuffle, I've added the missing links so should be hunky dory